app/assets/images/sephora_style_guide/svgs/order_status/awaiting-collection.svg

```diff
@@ -0,0 +1,7 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
```

Comment:
Do the svgs have `<title/>` and `<symbol/>` to it?
because front end is using `use{ 'xlink:href': "#icon-#{state_for_user.gsub('_', '-')}" }` to identify the icons

Reply:
Nope. Svgs here a bound to classes. So we can't exactly adopt the same idea as the sprite haml. The frontend will eventually have to use `:class='stateForUser'` for vue. Not sure if this answers your question?

Reply:
Is there an epic to convert all front end to vue? 

No issues adding the icons here. As long as the current pieces still stay in e-comm since it's still being used.

Reply:
https://sephora-asia.atlassian.net/browse/US-755